packages/yum/groonga-release.spec

```diff
@@ -41,7 +41,26 @@ Groonga release files
 %{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga
 %{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-groonga-RSA4096
 
+%post
+if [ -e "/etc/system-release" ]; then
```

Comment:
We can remove this with `if grep -q 'Amazon Linux release 2' /etc/system-release 2>/dev/null; then`.